pages/Documents/GettingStarted/customizing-the-conversational-cloud.md

```diff
@@ -18,32 +18,32 @@ indicator: both
 </div>
 
 <div class="card-container" style="margin-top:40px;">
-        <a class="welcome-card" target="_blank" href="/mobile-app-messaging-sdk-for-ios-overview.html"> 
+        <a class="welcome-card" target="_blank" href="/mobile-app-messaging-sdk-for-ios-overview.html">
```

Comment:
Nit/optional: I question these need a `target=_blank`—especially since the targets are on the dev site. Feel free to remove, otherwise let’s do that separately/later.